packages/at_secondary_server/lib/src/verb/handler/delete_verb_handler.dart

```diff
@@ -140,4 +149,23 @@ class DeleteVerbHandler extends ChangeVerbHandler {
         .build();
     NotificationManager.getInstance().notify(atNotification);
   }
+
+  List<String> _getProtectedKeys(String atsign) {
+    List<String> protectedKeys = [];
+    // fetch all protected private keys from config yaml
+    for (var key in AtSecondaryConfig.protectedKeys) {
+      // convert generic key name to actual public key
+      // of format: key:@atsign
+      protectedKeys.add('$key$atsign');
+    }
+    return protectedKeys;
+  }
+
+  bool _isProtectedKey(String key) {
+    if (protectedKeys!.contains(key)) {
```

Comment:
instead of contains..match the keys with the exact pattern
```
* public:signing_publickey<atsign>
* <atsign>:signing_privatekey<atsign>
* publickey<atsign>
* privatekey:at_pkam_publickey
```

Reply:
since this is a list, contains would match the entire key with protectedKeys